zebra/zebra_routemap.c

```diff
@@ -1065,6 +1064,37 @@ static struct route_map_rule_cmd route_match_ip_address_prefix_list_cmd = {
 	route_match_ip_address_prefix_list_compile,
 	route_match_ip_address_prefix_list_free};
 
+static route_map_result_t
```

Comment:
Why not make a generic function that both route_match_ip_address_prefix_list and route_match_ipv6_address_prefix_list call.  Instead of cut-n-pasting the same function effectively, call it something like route_match_address_prefix_list(void *rule, const struct prefix *prefix, route_map_object_t type, void *object, afi_t afi)